### PR TITLE
coverage report

### PR DESCRIPTION
**Pull request by agyen**

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -28,11 +28,19 @@ jobs:
       run: dotnet build --no-restore --configuration Release
     - name: Test
       run: dotnet test --verbosity normal -p:CollectCoverage=true -p:CoverletOutputFormat=opencover
+    - name: Current directory
+      run: dir
+    - name: Upload Test Report to Codecov
+      uses: codecov/codecov-action@v1
```
Comment (bingenito) [inline .github/workflows/continuous-integration.yml:34]: This is out of date and I believe decommissioned.  Update to @ 3

Comment (bingenito) [reply .github/workflows/continuous-integration.yml:34]: Confirmed that it was deprecated, see readme - https://github.com/codecov/codecov-action

Comment (agyen) [reply .github/workflows/continuous-integration.yml]: The latest I saw from the above link was v2 but I've updated it to v3 anyway.

Comment (bingenito) [reply .github/workflows/continuous-integration.yml]: https://github.com/codecov/codecov-action/releases

**Pull request merged**